go.mod

```diff
@@ -4,20 +4,23 @@ go 1.18
 
 require (
 	github.com/eapache/channels v1.1.0
+	github.com/facebookincubator/go-qfext v0.0.0-20210831195807-4cd18e79853b
```

Comment:
Seems like this and the cuckoo filter below are getting picked up unintentionally?

Reply:
thanks 👍